custom/icds_reports/views.py

```diff
@@ -364,10 +364,11 @@ def get(self, request, *args, **kwargs):
                 data = get_prevalence_of_undernutrition_sector_data(domain, config, loc_level, location, include_test)
             else:
                 data = get_prevalence_of_undernutrition_data_map(domain, config.copy(), loc_level, include_test)
-                sector = get_prevalence_of_undernutrition_sector_data(
-                    domain, config.copy(), loc_level, location, include_test
-                )
-                data.update(sector)
+                if loc_level == LocationTypes.BLOCK:
+                    sector = get_prevalence_of_undernutrition_sector_data(
+                        domain, config.copy(), loc_level, location, include_test
+                    )
+                    data.update(sector)
```

Comment:
just thinking out loud and may not be possible, but it'd be nice if there was some generic way to avoid having to make a change like this 10 times in the file...